README.md

```diff
@@ -14,7 +14,22 @@ With those dependencies, simply invoke `make` to build the document.
 
 Contributions are welcome to anyone who follows [our code of conduct](https://www.w3.org/Consortium/cepc/).
 
+When proposing new principles or updating existing ones, please take the [criteria](#criteria) into consideration.
 The editors will merge PRs after they have TAG consensus and will attempt to document that consensus
 by linking to the meeting minutes. If you notice a change that seems to have been merged without
 consensus, please comment on its PR or file an issue, and we'll either quickly find better evidence
 or back out the change until we can discuss it more fully.
+
+
+## Criteria
+
+The following criteria serve to guide the development of new and existing principles:
+
+* Grounded in patterns observed in specs, implementations, or web content.
+* Evidence-based.
+* Applicable across multiple technologies or use cases over time.
+* Based on definitions from open standards communities.
+* Expresses a clear value or priority e.g., prefer simplicity, preserve individual/user's agency, aligned with the Ethical Web Principles.
+* Logically consistent and compatible with other design principles and related guidelines (privacy, ethics, questionnaires, etc.).
+* Reflect general agreement within the TAG and broader community.
```

Comment:
No doubt that TAG has significant authority and responsibility to guide these principles and make final decisions. However, TAG is not infallible and must actively engage with the broader community. After all, the principles principles serve the entire web ecosystem. I think this is a balanced approach:

```suggestion
* Reflects TAG consensus, considers broader input, and is refined as needed in response to sustained objections.
```